src/index.html

```diff
@@ -6,9 +6,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__container">
+        <a class="logo__link" href="index.html">
+          <img class="logo__img" src="./images/logo.png"
+          alt="Moyo logo">
```

Comment:
```suggestion
          <img 
            class="logo__img" 
            src="./images/logo.png"
            alt="Moyo logo"
          >
```
Fix formatting everywhere
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/e81bc80b-9de3-4e82-b637-be360b55f6cf)

Reply:
Thanks for your comments!)